templates/data-sources/helm_charts.md.tmpl

```diff
@@ -0,0 +1,21 @@
+---
+Title: "Helm Charts Data Source"
+Description: |-
+    Read the Helm Charts from TMC.
+---
+
+# Helm Charts Repository
```

Comment:
Shouldn't this be Helm Repository Charts?

Reply:
Updated PTAL